codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
```

Comment:
Maybe you can also disable the comment in the PR: `comment: false`

Reply:
True, I'll add it. Thanks